### DISCUSSION TITLE
CSS as fastTags

### DISCUSSION THREAD

**Discussion by svarunid** [AnswerDotAI/fasthtml#282] [Ideas]
TLDR; Web components as FastTags with CSS styling.

After reading about fastHTML, the first thing I thought was that it was more like components in flutter but with HTML tags. 

I started my career as a flutter developer and I still do love it as a framework. One thing that I loved about flutter was its super easy way to design UI elements. 

I always used to struggle when it xomes to writing my own CSS. Tailwind rescued me back then but still everything was too much. Some properties were either directly or indirectly dependant on other properties for them to work. I find difficult to put all them together.

That's why I am thinking it would be a great way to develop a library that would provide CSS as fastTags.

**Comment by Viddesh1** [comment]
@svarunid ,

Yes, Fasthtml reminds me of flutter user interface style.
